pkg/cli/images.go

```diff
@@ -51,26 +74,105 @@ func (a *Image) Run(cmd *cobra.Command, args []string) error {
 		return strings.TrimPrefix(str, "sha256:")[:12]
 	})
 
-	var images []apiv1.Image
-	if len(args) == 1 {
-		img, err := c.ImageGet(cmd.Context(), args[0])
+	for _, image := range images {
+		if image.Tag == "" && image.Repository == "" && !a.All {
+			continue
+		}
+		out.Write(image)
+	}
+
+	return out.Err()
+}
+
+type ImageContainer struct {
```

Comment:
Can you unexport this (imageContainer) but keep the fields exported and see if it'll still work? I'm not 100% sure whether it will or not, but would be nice to keep it unexported if we can.

Reply:
Yep, agreed. It's only needed for looping through all of this.